apps/src/studio/blocks.js

```diff
@@ -1843,6 +1846,53 @@ exports.install = function (blockly, blockInstallOptions) {
     });
   };
 
+  blockly.Blocks.studio_setSpriteBehavior = {
+    helpUrl: '',
+    init: function () {
+      this.setHSV(184, 1.00, 0.74);
+      this.appendDummyInput()
+          .appendTitle(msg.setActor());
+      this.appendValueInput('SPRITE')
+          .setCheck(blockly.BlockValueType.NUMBER);
+      let hasTargetInput = true;
+      const behaviorValues = [
+        [msg.setSpriteChase(), BEHAVIOR_CHASE],
+        [msg.setSpriteFlee(), BEHAVIOR_FLEE],
+        [msg.toWander(), BEHAVIOR_WANDER],
+      ].map(kv => [kv[0], `"${kv[1]}"`]);
+      const behaviorDropdown = new blockly.FieldDropdown(behaviorValues, value => {
+        // Strip the quotes before comparing
+        value = value.substring(1, value.length - 1);
```

Comment:
it's not clear to me why we're wrapping the constants in quotes, only to immediately strip them here.